### DISCUSSION TITLE
Steam Year In Review 2023

### DISCUSSION THREAD

**Discussion by reanoff** [chr233/ASFEnhance#183] [功能建议 / Ideas]
REPLAY 2023

**Comment by woctezuma** [comment]
```
!replay ASF
```

- https://store.steampowered.com/yearinreview/
- https://steamcommunity.com/my/badges/66

**Reply by reanoff** [reply]
outputs the link 2022

**Reply by woctezuma** [reply]
Yes, but it unlocks the badge for 2023!

**Reply by woctezuma** [reply]
I have posted the issue at https://github.com/chr233/ASFEnhance/issues/184

**Comment by chr233** [comment]
fix in 2.0.5.0, the default year set to 2023, if given 2 or more arguments, the first will be treat as year, such as `REPLAY 2023 bot`
